src/components/internal-link/internal-link.js

```diff
@@ -38,7 +38,7 @@ const InternalLink = ({
   to,
   ...props
 }) => {
-  let location = { state: { eholdings: true } };
+  let location = { state: { ...to.state, eholdings: true, } };
```

Comment:
Can we get location state from `useLocation` in this component, instead of passing it as a prop?

Reply:
The `state` is not yet in the `location`, we just want to transfer it there by clicking on this link.